app/src/main/res/drawable/context_menu_button_background.xml

```diff
@@ -2,10 +2,16 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<shape
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    android:shape="oval">
-    <solid android:color="@color/default_background_gray" />
-    <stroke android:color="@color/overlay_button_border"
-        android:width="0.5dp" />
-</shape>
\ No newline at end of file
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_pressed="true">
+        <shape android:shape="oval">
+            <solid android:color="#5b5b5c" />
```

Comment:
Ideally this was the same pressed color as the overlay view but I can see why it's not (because we use alpha for the overlay).

Reply:
Yeah, we wanted the same visual effect but that was hard to achieve with one overlay as this left the bounds of the home_tile.